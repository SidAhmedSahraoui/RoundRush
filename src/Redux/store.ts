import { Action, ThunkAction, configureStore } from "@reduxjs/toolkit";

import auth from "./auth/auth-slice";


export const store = configureStore({
  reducer: {
    auth
  },
  devTools: true,
});

export type AppDispatch = typeof store.dispatch;
export type RootState = ReturnType<typeof store.getState>;
export type AppThunk<ReturnType = void> = ThunkAction<
  ReturnType,
  RootState,
  unknown,
  Action<string>
>;
