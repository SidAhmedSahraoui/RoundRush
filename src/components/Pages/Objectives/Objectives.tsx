import React, { Fragment, useState } from "react";
import useStyles from "./style";
import Bell from "../../../img/bell.svg";
import Add from "../../../img/add.svg";
import Slack from "../../../img/slack.svg";
import Calendar from "../../../img/calendar.svg";
import { RocketLaunch, ArrowUpward } from "@mui/icons-material";
import Conference from "../../../img/conference.svg";
import Checkbox from '@mui/material/Checkbox';
import Paper from "@mui/material/Paper";
import InputBase from "@mui/material/InputBase";
import { Icon } from "@mui/material";
import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
import SearchIcon from "@mui/icons-material/Search";
import { Link } from "react-router-dom";
import Oval from "../../../img/Oval.svg";
import Oval4 from "../../../img/Oval4.svg";
import Plus from "../../../img/plus.svg"
import Close from "../../../img/close.svg";
import Delete from "../../../img/delete.svg";
import High from "../../../img/high.svg";
import Copy from "../../../img/copy.svg";
import { v4 as uuid } from "uuid";
import Box from '@mui/material/Box';
import Modal from '@mui/material/Modal';
import { oProps } from "../../../types/types";

const Objectives: React.FC<oProps> = ({objs}) => {
  const classes = useStyles();
  const [dis, setDis] = useState(false);
  const clickHandler = (event: React.MouseEvent<HTMLImageElement>) => {
    event.preventDefault();
    setDis((dis) => !dis);
    console.log(dis)
  };    
  return (
    <>
      <main className={classes.main}>
        <nav>
          <section>
            <h1>Objectives</h1>
            <img src={Conference} alt="bell" />
          </section>

          <div className="icons">
            <div className="iter">
              <h2>
                iteration type : <strong>ROUND</strong>
              </h2>
            </div>
            <img src={Slack} alt="bell" />
            <img src={Calendar} alt="bell" />
            <button>
              <Icon sx={{ color: "#FFFFFF" }} component={RocketLaunch} />
              <h3>Upgrade</h3>
            </button>
            <img onClick={clickHandler} style={{cursor: "pointer"}} src={Bell} alt="bell" />
            <img src={Add} alt="add" />
          </div>
        </nav>
        <span></span>
        <section className="search">
          <div className="left">
            <Paper
              component="form"
              sx={{
                borderRadius: "4px",
                backgroundColor: "#ffffff",
                boxShadow: "none",
                fontSize: "14px",
                fontWeight: 400,
                display: "flex",
                justifyContent: "space-evenly",
                alignItems: "center",
                width: "190px",
                height: "32px",
                padding: "4px",
                border: `2px solid rgba(76, 132, 255)`,
              }}>
              <InputBase
                type="text"
                placeholder="Choose something"
                sx={{
                  width: "80%",
                }}
              />
              <SearchIcon sx={{ color: "rgba(76, 132, 255)" }} />
            </Paper>
            <Icon className="icon" component={ArrowUpward} />
          </div>
          <div className="right">
            <button>Milestones</button>
            <Link className="link" to="/objectives">
              Requirements
            </Link>
          </div>
        </section>
        <main className="core">
          <div className="line">
            <div className="it name">NAME</div>
            <div className="it prio">PRIORITY</div>
            <div className="it status">STATUS</div>
            <div className="it assi">ASSIGNEES</div>
            <div className="it date">DUE DATE</div>
            <div className="it check">CHECKMARKS</div>
          </div>
          {objs.map((obj) => (
            <Fragment key={uuid()}>
              <div className="_line">
                <div className="_it _name">{obj.name}</div>
                <div className="_it _prio">
                  <div
                    className="cont"
                    style={{
                      background:
                        obj.priority === 1
                          ? "#DB54FD"
                          : obj.priority === 2
                          ? "#FD5461"
                          : obj.priority === 3
                          ? "#FFAB2A"
                          : "#29C293",
                    }}>
                    {obj.priority === 1
                      ? "URGENT"
                      : obj.priority === 2
                      ? "HIGH"
                      : obj.priority === 3
                      ? "MEDUIM"
                      : "LOW"}
                  </div>
                </div>
                <div className="_it _status">
                  <div className="cont">Not Done</div>
                </div>
                <div className="_it _assi">
                  <div className="cont">
                    <img
                      style={{ width: "20px", height: "20px", zIndex: "3" }}
                      className="img1"
                      src={Oval}
                      alt=""
                    />
                    <img
                      style={{ width: "20px", height: "20px", zIndex: "4" }}
                      src={Oval}
                      alt=""
                    />
                    <img
                      style={{ width: "20px", height: "20px", zIndex: "5" }}
                      className="img2"
                      src={Oval}
                      alt=""
                    />
                  </div>
                </div>
                <div className="_it _date">{obj.date}</div>
                <div className="_it _check"> {obj.check} </div>
              </div>
              <span></span>
            </Fragment>
          ))}
        </main>
      </main>
      <Modal
        open={dis}
        onClose={clickHandler}
        sx={{border: "none"}}
        
      >
        <Box className={classes.second} >
        <nav>
          <img onClick={clickHandler} src={Delete} alt="" />
          <img onClick={clickHandler} src={Close} alt="" />
        </nav>
        <div className="core">
          <h1>Notifications Center</h1>
          <h5>Description</h5>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam
            accusamus exercitationem rem provident laborum natus pariatur minima
            veritatis doloremque rerum temporibus culpa nulla fugiat libero,
            incidunt ratione excepturi aliquam voluptas.
          </p>
          <h6>Priority</h6>
          <div className="prio">
            <img src={High} alt="" />
            <h3>High Priority</h3>
          </div>
          <h6>Subproject</h6>
          <div className="prio">
            <img src={Copy} alt="" />
            <h3>Webapp</h3>
          </div>
          <h6 style={{ marginLeft: "32px" }}>Assignees</h6>
          <div className="assi">
            <div className="rec">
              <img src={Oval4} alt="" />
              <h3>lene Robert</h3>
              <img src={Close} alt="" />
            </div>
            <div className="rec">
              <img src={Oval4} alt="" />
              <h3>lene Robert</h3>
              <img src={Close} alt="" />
            </div>
          </div>
          <h6>Linked Projects</h6>
          <h3 className="date">
            12/12/2022
          </h3>
          <h6>Due Date</h6>
          <h3 className="date">
            12/12/2022
          </h3>
          <div className="title">
            <h1>CHECKMARKS</h1>
            <img src={Plus} alt="" />
          </div>
          <div className="rec">
            <h1>Notifications Service</h1>
            <div>
            <MoreHorizIcon sx={{color: "#ADB0B8"}}  />
            <Checkbox inputProps={{ 'aria-label': 'Checkbox demo' }}  defaultChecked />
            </div>
          </div>
          <div className="rec">
            <h1>Notifications Service</h1>
            <div>
            <MoreHorizIcon sx={{color: "#ADB0B8"}}  />
            <Checkbox inputProps={{ 'aria-label': 'Checkbox demo' }}  defaultChecked />
            </div>
          </div>
          <div className="rec">
            <h1>API server</h1>
            <div>
            <MoreHorizIcon sx={{color: "#ADB0B8"}} />
            <Checkbox inputProps={{ 'aria-label': 'Checkbox demo' }}  defaultChecked />
            </div>
          </div>
        </div>
        </Box>
      </Modal>
    </>
  );
};
export default Objectives;
