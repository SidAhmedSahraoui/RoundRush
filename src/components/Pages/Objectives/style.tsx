import { createUseStyles } from "react-jss";

const useStyles = createUseStyles({
  second: {
    width: "500px",
    position: "absolute",
    overflowY: "scroll",
    right: "0px",
    top: "0px",
    background: "#ffffff",
    border: "2px solid #E8F0FE",
    height: "100%",
    zIndex: 8,
    transition:" 0.4s all ease",
    "&:focus": {
      border: "2px solid #E8F0FE",
    },

    "&::-webkit-scrollbar": {
      width: "5px",
    },

    "::-webkit-scrollbar-track": {
      background: "#fff",
    },

    "::-webkit-scrollbar-thumb": {
      background: "#c2c2c2",
      borderRadius: "10px",
    },

    "&::-webkit-scrollbar-thumb:hover": {
      background: "#919090",
    },
    "& nav": {
      height: "50px",
      borderBottom: "2px solid #E8F0FE",
      display: "flex",
      justifyContent: "flex-end",
      alignItems: "center",
      "& img": {
        margin: "auto 12px",
        cursor: "pointer"
      },
    },
    "& .core": {
      "& h1": {
        fontWeight: 600,
        fontSize: "24px",
        fontFamily: "Rubik",
        margin: "24px",
        textAlign: "left",
        color: "rgb(21, 27, 38)",
      },
      "& h5": {
        fontWeight: 400,
        fontSize: "12px",
        marginLeft: "32px",
        fontFamily: "Rubik",
        textAlign: "left",
        color: " #ADB0B8",
      },
      "& p": {
        width: "390px",
        color: "rgba(21, 27, 38, 0.5)",
        fontWeight: 400,
        fontSize: "12px",
        textAlign: "left",
        marginLeft: "32px",
        fontFamily: "Rubik",
        marginTop: "14px",
      },
      "& h6": {
        color: "#ADB0B8",
        fontWeight: 400,
        fontSize: "12px",
        textAlign: "left",
        marginLeft: "24px",
        fontFamily: "Rubik",
        marginTop: "24px",
      },
      "& .prio": {
        width: "120px",
        alignItems: "center",
        display: "flex",
        justifyContent: "space-evenly",
        marginLeft: "32px",
        marginTop: "14px",
        "& h3": {
          color: "#31394E",
          fontWeight: 400,
          fontSize: "14px",
          fontFamily: "Rubik",
        },
        "& img": {},
      },
      "& .date": {
        color: "#31394E",
        fontWeight: 400,
        fontSize: "14px",
        fontFamily: "Rubik",
        textAlign: "left",
        margin: "14px 0px 30px 32px",
      },
      "& .title": {
        width: "90%",
        margin: "33px 0px 0px 33px",
        display: "flex",
        justifyContent: "space-between",
        "& h1": {
          color: "#4C84FF",
          fontWeight: 600,
          margin: "0px",
          fontSize: "12px",
          fontFamily: "Rubik",
        },
      },
      "& .assi": {
        margin: "14px 32px",
        display: "flex",
        "& .rec": {
          background: "rgba(0, 121, 255, 0.2963)",
          borderRadius: "7.5px",
          marginRight: "8px",
          width: "160px",
          height: "36px",
          display: "flex",
          alignItems: "center",
          justifyContent: "space-evenly",
          cursor: "pointer",
          "& h3": {
            color: "#0079FF",
            fontWeight: 400,
            fontSize: "12.5px",
            fontFamily: "Rubik",
          },
        },
      },
      "& .rec": {
        width: "100%",
        height: "50px",
        display: "flex",
        justifyContent: "space-between",
        alignItems: "center",
        "&:hover": {
          background: "#E8F0FE",
          cursor: "pointer"
        },
        "& div": {
          display: "flex",
          alignItems: "center",
          width: "20%",
          justifyContent: "space-evenly",
        },
        "& h1": {
          color: "#000000",
          fontSize: "14px",
          fontWeight: 400,
          fontFamily: "Rubik",
        },
      },
    },
  },
  main: {
    width: "80%",
    margin: "30px auto",
    "@media screen and (max-width: 1268px)": {
      width: "90%",
    },
    "& nav": {
      display: "flex",
      justifyContent: "space-between",
      "& section": {
        display: "flex",
        justifyContent: "space-between",
        width: "160px",
        "& h1": {
          color: "#31394E",
          fontSize: "24px",
          fontWeight: 400,
          fontFamily: "Rubik",
        },
      },
      "& .icons": {
        width: "55%",
        display: "flex",
        justifyContent: "space-between",
        "& .iter": {
          width: "250px",
          height: "32px",
          display: "flex",
          justifyContent: "center",
          alignItems: "center",
          borderRadius: "4px",
          backgroundColor: "#E8F0FE",
          "& h2": {
            fontSize: "14px",
            fontWeight: 400,
            fontFamily: "Rubik",
            color: "#4C84FF",
          },
        },
        "& button": {
          width: "130px",
          height: "32px",
          border: "none",
          borderRadius: "6px",
          backgroundColor: "#29C293",
          display: "flex",
          justifyContent: "space-evenly",
          alignItems: "center",
          cursor: "pointer",
          "& h3": {
            color: "#FFFFFF",
            fontSize: "14px",
            fontWeight: 600,
            fontFamily: "Rubik",
          },
        },
      },
    },
    "& span": {
      display: "block",
      width: "100%",
      marginTop: "5px",
      margin: "auto",
      height: "2px",
      background: "#F4F6FC",
    },
    "& .search": {
      marginTop: "12px",
      width: "100%",
      display: "flex",
      justifyContent: "space-between",
      alignItems: "center",
      "& .left": {
        width: "30%",
        display: "flex",
        alignItems: "center",
        "& .icon": {
          color: " rgba(49, 57, 78, 0.4)",
        },
      },
      "& .right": {
        width: "30%",
        display: "flex",
        justifyContent: "space-evenly",
        alignItems: "center",
        "& button": {
          border: "2px solid rgb(76, 132,255)",
          color: "rgb(76, 132,255)",
          height: "32px",
          width: "129px",
          background: "#ffffff",
          borderRadius: "6px",
          fontSize: "14px",
          cursor: "pointer",
          fontWeight: 600,
          fontFamily: "Rubik",
        },
        "& .link": {
          fontSize: "14px",
          fontWeight: 600,
          fontFamily: "Rubik",
          color: "#ADB0B8",
          textDecoration: "none",
        },
      },
    },
    "& .core": {
      "& .line": {
        display: "flex",
        justifyContent: "flex-start",
        alignContent: "start",
        margin: "35px auto",
        "& .it": {
          fontSize: "14px",
          fontWeight: 600,
          fontFamily: "Rubik",
          color: "rgba(49, 57, 78, 0.5)",
          textAlign: "center",
          width: "10%",
        },
        "& .name": {
          width: "50%",
          textAlign: "left",
        },
      },
      "& ._line": {
        display: "flex",
        justifyContent: "flex-start",
        alignContent: "start",
        margin: "15px auto",
        "& ._it": {
          textAlign: "center",
          width: "10%",
          display: "flex",
          justifyContent: "center",
        },
        "& ._name": {
          display: "flex",
          textAlign: "left",
          alignItems: "center",
          justifyContent: "flex-start",
          width: "50%",
          fontSize: "14px",
          fontWeight: 400,
          fontFamily: "Rubik",
          color: "#31394E",
        },
        "& ._prio": {
          "& .cont": {
            height: "32px",
            width: "80px",
            display: "flex",
            justifyContent: "center",
            textAlign: "center",
            alignItems: "center",
            borderRadius: "6px",
            color: "#FFFFFF",
            fontSize: "12px",
            fontWeight: 400,
            fontFamily: "Rubik",
          },
        },
        "& ._status": {
          "& .cont": {
            height: "32px",
            width: "80px",
            display: "flex",
            justifyContent: "center",
            textAlign: "center",
            alignItems: "center",
            borderRadius: "6px",
            color: "#FFFFFF",
            fontSize: "12px",
            fontWeight: 400,
            fontFamily: "Rubik",
            background: "#D6D7DC",
          },
        },
        "& ._date": {
          color: "rgba(49, 57, 78, 0.5)",
          fontSize: "12px",
          fontWeight: 400,
          fontFamily: "Rubik",
          display: "flex",
          justifyContent: "center",
          textAlign: "center",
          alignItems: "center",
        },
        "& ._check": {
          color: "rgba(49, 57, 78, 0.5)",
          fontSize: "12px",
          fontWeight: 400,
          fontFamily: "Rubik",
          display: "flex",
          justifyContent: "center",
          textAlign: "center",
          alignItems: "center",
        },
        "& ._assi": {
          "& .cont": {
            display: "flex",
            justifyContent: "flex-end",
            "& .img1": {
              position: "relative",
              left: "8px",
            },
            "& .img2": {
              position: "relative",
              right: "8px",
            },
          },
        },
      },
    },
  },
});
export default useStyles;
