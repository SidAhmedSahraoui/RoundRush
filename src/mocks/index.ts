import { Task, Todo, Objective, userObject } from "../types/types";

export const tasks: Array<Task> = [
  {
    name: "LD Website",
    code: "LW",
    color: "#DB54FD",
  },
  {
    name: "LD Marketing",
    code: "LM",
    color: "#61D26F",
  },
  {
    name: "RoundRush",
    code: "R",
    color: "#1FAAE9",
  },
  {
    name: "RoundRush Website",
    code: "RW",
    color: "#4086E0",
  },
  {
    name: "DevOps",
    code: "DO",
    color: "#EEC301",
  },
  {
    name: "MicroServices",
    code: "MS",
    color: "#FD612C",
  },
  {
    name: "Deep Learning",
    code: "DL",
    color: "#FD612C",
  },
  {
    name: "SpringBoot",
    code: "SP",
    color: "#1FAAE9",
  },
  {
    name: "Java",
    code: "J",
    color: "#61D26F",
  },
];
export const todos: Array<Todo> = [
    {
        priorety: 4,
        type: "task",
        code: "VMS-20",
        title: "Certe, inquam, pertinax non intell...",
        parent: "LD Website",
        color: "#61D26F",
      },
      {
        priorety: 4,
        type: "task2",
        code: "VMS-11",
        title: "Ac hac et ac tristique luctus.",
        parent: "Roundrush",
        color: "#1FAAE9",
      },
      {
        priorety: 3,
        type: "macro",
        code: "VMS-28",
        title: "Magna bibendum nibh mi habitant.",
        parent: "Marketing",
        color: "#1FAAE9",
      },
      {
        priorety: 3,
        type: "task",
        code: "VMS-22",
        title: "Ac hac et ac tristique luctus.",
        parent: "LD Website",
        color: "#61D26F",
      },
      {
        priorety: 2,
        type: "task2",
        code: "VMS-11",
        title: "Varius ultricies vitae eget.",
        parent: "LD Website",
        color: "#7A6FEF",
      },
      {
        priorety: 1,
        type: "macro",
        code: "VMS-22",
        title: "Certe, inquam, pertinax non intell...",
        parent: "LD Website",
        color: "#7A6FEF",
      },
];
export const objs: Array<Objective> = [
    {
        name: "Notifications Center",
        priority: 4,
        status: false,
        date: "15/01/2020",
        check: "5/10",
      },
      {
        name: "App Mobile",
        priority: 3,
        status: false,
        date: "15/01/2020",
        check: "5/10",
      },
      {
        name: "DevOps",
        priority: 2,
        status: false,
        date: "15/01/2020",
        check: "5/10",
      },
      {
        name: "Kubernetes",
        priority: 1,
        status: false,
        date: "15/01/2020",
        check: "5/10",
      },
      {
        name: "Docker",
        priority: 4,
        status: false,
        date: "15/01/2020",
        check: "5/10",
      },
]

export const users: Array<userObject> = [
  {
    name: "Mario Panariello",
    owner: true,
    email: "mario.panariello@lastingdynamics.com",
    active: true,
  },
  {
    name: "Brandon Watson",
    owner: false,
    email: "mario.panariello@lastingdynamics.com",
    active: true,
  },
  {
    name: "Norma Copper",
    owner: false,
    email: "mario.panariello@lastingdynamics.com",
    active: false,
  },
  {
    name: "Mario Panariello",
    owner: false,
    email: "mario.panariello@lastingdynamics.com",
    active: false,
  },
  {
      name: "Mario Panariello",
      owner: true,
      email: "mario.panariello@lastingdynamics.com",
      active: true,
    }
]