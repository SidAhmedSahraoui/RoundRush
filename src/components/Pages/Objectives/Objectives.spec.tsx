/**
 * @jest-environment jsdom
 */

import React from "react";
import { cleanup, render } from "@testing-library/react";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import { store } from "../../../redux/store";
import Objectives from "./Objectives";
import { objs } from "../../../mocks";

describe("Objectives", () => {
  afterEach(() => cleanup());

  test("Render Objectives with Data from Mock", () => {
    const view = render(
      <Provider store={store}>
        <Router>
          <Objectives objs={objs} />
        </Router>
      </Provider>
    );

    expect(view.container).toMatchSnapshot();
  });
});
