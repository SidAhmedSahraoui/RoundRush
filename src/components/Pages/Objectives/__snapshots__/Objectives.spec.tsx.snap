// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Objectives Render Objectives with Data from Mock 1`] = `
<div>
  <main
    class="main-0-2-2"
  >
    <nav>
      <section>
        <h1>
          Objectives
        </h1>
        <img
          alt="bell"
          src="conference.svg"
        />
      </section>
      <div
        class="icons"
      >
        <div
          class="iter"
        >
          <h2>
            iteration type : 
            <strong>
              ROUND
            </strong>
          </h2>
        </div>
        <img
          alt="bell"
          src="slack.svg"
        />
        <img
          alt="bell"
          src="calendar.svg"
        />
        <button>
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium css-zqw3rs-MuiSvgIcon-root-MuiIcon-root"
            data-testid="RocketLaunchIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M9.19 6.35c-2.04 2.29-3.44 5.58-3.57 5.89L2 10.69l4.05-4.05c.47-.47 1.15-.68 1.81-.55l1.33.26zM11.17 17s3.74-1.55 5.89-3.7c5.4-5.4 4.5-9.62 4.21-10.57-.95-.3-5.17-1.19-10.57 4.21C8.55 9.09 7 12.83 7 12.83L11.17 17zm6.48-2.19c-2.29 2.04-5.58 3.44-5.89 3.57L13.31 22l4.05-4.05c.47-.47.68-1.15.55-1.81l-.26-1.33zM9 18c0 .83-.34 1.58-.88 2.12C6.94 21.3 2 22 2 22s.7-4.94 1.88-6.12C4.42 15.34 5.17 15 6 15c1.66 0 3 1.34 3 3zm4-9c0-1.1.9-2 2-2s2 .9 2 2-.9 2-2 2-2-.9-2-2z"
            />
          </svg>
          <h3>
            Upgrade
          </h3>
        </button>
        <img
          alt="bell"
          src="bell.svg"
          style="cursor: pointer;"
        />
        <img
          alt="add"
          src="add.svg"
        />
      </div>
    </nav>
    <span />
    <section
      class="search"
    >
      <div
        class="left"
      >
        <form
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1x91vgv-MuiPaper-root"
        >
          <div
            class="MuiInputBase-root MuiInputBase-colorPrimary css-9qawp9-MuiInputBase-root"
          >
            <input
              class="MuiInputBase-input css-yz9k0d-MuiInputBase-input"
              placeholder="Choose something"
              type="text"
              value=""
            />
          </div>
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-ixemat-MuiSvgIcon-root"
            data-testid="SearchIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
          </svg>
        </form>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium icon css-1n7dniw-MuiSvgIcon-root-MuiIcon-root"
          data-testid="ArrowUpwardIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="m4 12 1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
          />
        </svg>
      </div>
      <div
        class="right"
      >
        <button>
          Milestones
        </button>
        <a
          class="link"
          href="/objectives"
        >
          Requirements
        </a>
      </div>
    </section>
    <main
      class="core"
    >
      <div
        class="line"
      >
        <div
          class="it name"
        >
          NAME
        </div>
        <div
          class="it prio"
        >
          PRIORITY
        </div>
        <div
          class="it status"
        >
          STATUS
        </div>
        <div
          class="it assi"
        >
          ASSIGNEES
        </div>
        <div
          class="it date"
        >
          DUE DATE
        </div>
        <div
          class="it check"
        >
          CHECKMARKS
        </div>
      </div>
      <div
        class="_line"
      >
        <div
          class="_it _name"
        >
          Notifications Center
        </div>
        <div
          class="_it _prio"
        >
          <div
            class="cont"
            style="background: rgb(41, 194, 147);"
          >
            LOW
          </div>
        </div>
        <div
          class="_it _status"
        >
          <div
            class="cont"
          >
            Not Done
          </div>
        </div>
        <div
          class="_it _assi"
        >
          <div
            class="cont"
          >
            <img
              alt=""
              class="img1"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 3;"
            />
            <img
              alt=""
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 4;"
            />
            <img
              alt=""
              class="img2"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 5;"
            />
          </div>
        </div>
        <div
          class="_it _date"
        >
          15/01/2020
        </div>
        <div
          class="_it _check"
        >
           
          5/10
           
        </div>
      </div>
      <span />
      <div
        class="_line"
      >
        <div
          class="_it _name"
        >
          App Mobile
        </div>
        <div
          class="_it _prio"
        >
          <div
            class="cont"
            style="background: rgb(255, 171, 42);"
          >
            MEDUIM
          </div>
        </div>
        <div
          class="_it _status"
        >
          <div
            class="cont"
          >
            Not Done
          </div>
        </div>
        <div
          class="_it _assi"
        >
          <div
            class="cont"
          >
            <img
              alt=""
              class="img1"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 3;"
            />
            <img
              alt=""
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 4;"
            />
            <img
              alt=""
              class="img2"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 5;"
            />
          </div>
        </div>
        <div
          class="_it _date"
        >
          15/01/2020
        </div>
        <div
          class="_it _check"
        >
           
          5/10
           
        </div>
      </div>
      <span />
      <div
        class="_line"
      >
        <div
          class="_it _name"
        >
          DevOps
        </div>
        <div
          class="_it _prio"
        >
          <div
            class="cont"
            style="background: rgb(253, 84, 97);"
          >
            HIGH
          </div>
        </div>
        <div
          class="_it _status"
        >
          <div
            class="cont"
          >
            Not Done
          </div>
        </div>
        <div
          class="_it _assi"
        >
          <div
            class="cont"
          >
            <img
              alt=""
              class="img1"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 3;"
            />
            <img
              alt=""
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 4;"
            />
            <img
              alt=""
              class="img2"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 5;"
            />
          </div>
        </div>
        <div
          class="_it _date"
        >
          15/01/2020
        </div>
        <div
          class="_it _check"
        >
           
          5/10
           
        </div>
      </div>
      <span />
      <div
        class="_line"
      >
        <div
          class="_it _name"
        >
          Kubernetes
        </div>
        <div
          class="_it _prio"
        >
          <div
            class="cont"
            style="background: rgb(219, 84, 253);"
          >
            URGENT
          </div>
        </div>
        <div
          class="_it _status"
        >
          <div
            class="cont"
          >
            Not Done
          </div>
        </div>
        <div
          class="_it _assi"
        >
          <div
            class="cont"
          >
            <img
              alt=""
              class="img1"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 3;"
            />
            <img
              alt=""
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 4;"
            />
            <img
              alt=""
              class="img2"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 5;"
            />
          </div>
        </div>
        <div
          class="_it _date"
        >
          15/01/2020
        </div>
        <div
          class="_it _check"
        >
           
          5/10
           
        </div>
      </div>
      <span />
      <div
        class="_line"
      >
        <div
          class="_it _name"
        >
          Docker
        </div>
        <div
          class="_it _prio"
        >
          <div
            class="cont"
            style="background: rgb(41, 194, 147);"
          >
            LOW
          </div>
        </div>
        <div
          class="_it _status"
        >
          <div
            class="cont"
          >
            Not Done
          </div>
        </div>
        <div
          class="_it _assi"
        >
          <div
            class="cont"
          >
            <img
              alt=""
              class="img1"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 3;"
            />
            <img
              alt=""
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 4;"
            />
            <img
              alt=""
              class="img2"
              src="Oval.svg"
              style="width: 20px; height: 20px; z-index: 5;"
            />
          </div>
        </div>
        <div
          class="_it _date"
        >
          15/01/2020
        </div>
        <div
          class="_it _check"
        >
           
          5/10
           
        </div>
      </div>
      <span />
    </main>
  </main>
</div>
`;
